hardware-and-software-requirements.md

```diff
@@ -26,9 +24,29 @@ As an open source distributed NewSQL database with high performance, TiDB can be
 > - A large number of TiDB tests have been run on the CentOS 7.3 system, and in our community there are a lot of best practices in which TiDB is deployed on the Linux operating system. Therefore, it is recommended to deploy TiDB on CentOS 7.3 or later.
 > - The support for the Linux operating systems above includes the deployment and operation in physical servers as well as in major virtualized environments like VMware, KVM and XEN.
 
+## Software recommendations
+
+### Control Machine
+
+| Software | Version |
+| :--- | :--- |
+| sshpass | 1.06 or later |
+| TiUP | 0.6.2 or later |
+
+> **Note:**
+>
+> It is required that you [deploy TiUP](/tiup/tiup-cluster.md) on the Control Machine to operate and manage TiDB clusters.
```

Comment:
```suggestion
> It is required that you [deploy TiUP](/production-deployment-using-tiup.md#第-2-步在中控机上安装-tiup-组件) on the Control Machine to operate and manage TiDB clusters.
```
Waiting on https://github.com/pingcap/docs-cn/pull/3228/files#r433695667

Reply:
Please see https://github.com/pingcap/docs-cn/pull/3228/files#r433772498 to fix the docs. @ran-huang